tests/test_forward_sampling.py

```diff
@@ -8,7 +8,7 @@
 from pymc4.flow.executor import EvaluationError
 
 
-@pytest.fixture(scope="module", params=[(), (1,), (2,), (1, 1), (7, 3)], ids=str)
+@pytest.fixture(scope="module", params=[(1, 10), (1, 1), (1, 2), (1, 1), (3, 7)], ids=str)
```

Comment:
Why did you remove the `()` sample shape? I think it's important to test it to see if we get the correct core shape or for some buggy reason, we are inventing new dimension